ui/app/manager/src/components/configuration/or-conf-realm/or-conf-realm-card.ts

```diff
@@ -274,6 +277,16 @@ export class OrConfRealmCard extends LitElement {
     render() {
         const colors = this._getColors();
         const app = this;
+        const languageNames = ISO6391.getAllNames();
+        const languages: string[][] = ISO6391.getAllCodes()
+            .map((code, index) => ([code, languageNames[index]]))
+            .sort((a, b) => a[1].localeCompare(b[1]));
```

Comment:
This mapping an sorting looks like expensive string operations to do everytime the component is rendered. Wouldn't it make sense to cache the result and use that?